src/display/core/Controller.cpp

```diff
@@ -424,3 +426,73 @@ void Controller::onVolumetricMeasurement(double measurement) const {
         lastProcess->updateVolume(measurement);
     }
 }
+
+void Controller::handleBrewButton(int brewButtonStatus) {
+    printf("current screen %d, brew button %d\n", getMode(), brewButtonStatus);
+    if(brewButtonStatus == 1) {
+        switch (getMode())
+        {
+        case MODE_STANDBY:
+            deactivateStandby();
+            break;
+        case MODE_BREW:
+            if(!isActive()) {
+                deactivateStandby();
+                clear();
+                activate();
+            }
+            break;
+        case MODE_WATER:
+            activate();
+            break;
+        default:
+            break;
+        }
+    } else {
+        switch (getMode())
+        {
+        case MODE_BREW:
+            if(isActive()) {
```

Comment:
Maybe we can clear the "finished screen" by unflipping the brew button?

Reply:
As another thing. I know some people want to use momentary buttons instead of the toggle buttons. Maybe we could add a setting for momentary buttons?